requirements/main.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/main.txt requirements/main.in
+#    pip-compile --allow-unsafe --config=pyproject.toml --generate-hashes --output-file=requirements/main.txt requirements/main.in
```

Comment:
```suggestion
#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/main.txt requirements/main.in
```
I think we need to do a bigger change to our lockfiles which will include this.

Reply:
Makes sense -- that got added when I ran `make requirements/main.txt`, so that target might need to be tweaked slightly in the mean time.